scripts/start-test-setup

```diff
@@ -8,6 +8,6 @@ NODE=`which node`
 $NODE ./scripts/addCandidateBalances.js
 for i in $(seq 0 6); do
     rm -rf -- "./parity-data/node$i/chains"
-    "$PARITY" --config "./config/node${i}.toml" > "./parity-data/node${i}/log" 2>&1 &
+    "$PARITY" --interface=local --nat=none --config "./config/node${i}.toml" > "./parity-data/node${i}/log" 2>&1 &
```

Comment:
Do you need the `--nat` option given that we have it in the `toml` files? https://github.com/poanetwork/posdao-test-setup/blob/e003aed8621fea3fd1441cf3689d911bf4973e85/config/node0.toml#L11

Reply:
You're right; I removed both from the script and added `network = "local"` to the config files.